src/special/webpack.js

```diff
@@ -39,14 +42,58 @@ function getLoaders(deps, loaders) {
     .value();
 }
 
+function parseWebpack1(module, deps) {
+  const loaders = getLoaders(deps, module.loaders);
+  const preLoaders = getLoaders(deps, module.preLoaders);
+  const postLoaders = getLoaders(deps, module.postLoaders);
+  return [...loaders, ...preLoaders, ...postLoaders];
+}
+
+function mapRuleLoaders(module) {
+  return module.rules
+    .filter(rule => rule.loaders)
+    .map(rule => rule.loaders.map(loader => ({
+      loader,
+    })));
+}
+
+function mapRuleUse(module) {
+  return module.rules
+    // filter use or loader (loader is a shortcut to use)
+    .filter(rule => rule.use || rule.loader)
+    .map((rule) => {
+      const key = rule.use ? 'use' : 'loader';
+      const coercedArray = [].concat(rule[key]);
+
+      // if it's an array, apply the two rules above for each element
+      return coercedArray.map((value) => {
+        if (typeof value === 'object') {
+          return value;
+        }
+
+        return {
+          loader: value,
```

Comment:
If we check `typeof item === 'string'` in `extractLoaders`, this could be simplify too.